css/metabox-320.css

```diff
@@ -170,7 +170,7 @@
 .snippet_container .title {
 	display: block;
 	overflow: hidden;
-	width: 600px;
+	width: 512px;
```

Comment:
My guess is this should be 600px: see https://github.com/Yoast/wordpress-seo/pull/4729